src/www/views/servers_view/server_connection_indicator/index.ts

```diff
@@ -110,7 +118,7 @@ export class ServerConnectionIndicator extends LitElement {
       /* prettier-ignore */
       circleSize => css`
         .circle-${circleSize} {
-          transform: var(--circle-${circleSize}-scale);
+          transform: translate3d(0, 0, 0) var(--circle-${circleSize}-scale);
```

Comment:
Nowadays we can use `will-change` but this was the way to trigger GPU support on older browsers back in the day (still today)

Reply:
I suppose I could add a `will-change` for futureproofing, no?